pkg/utils/etcdutil/etcdutil.go

```diff
@@ -119,12 +120,17 @@ func ListEtcdMembers(ctx context.Context, client *clientv3.Client) (*clientv3.Me
 		time.Sleep(time.Duration(d) * time.Second)
 	})
 	newCtx, cancel := context.WithTimeout(ctx, DefaultRequestTimeout)
-	listResp, err := client.MemberList(newCtx)
+	// After the etcd server is upgraded to v3.5.0, the MemberList API will return the member list in a linearizable way by default.
+	// It is introduced by https://github.com/etcd-io/etcd/pull/11639
+	// If Linearizable is set to false, the member list will be returned with server's local data.
+	// If Linearizable is set to true, it is served with linearizable guarantee. If the server is disconnected from quorum, `MemberList` call will fail.
+	c := clientv3.RetryClusterClient(client)
+	resp, err := c.MemberList(newCtx, &etcdserverpb.MemberListRequest{Linearizable: false})
```

Comment:
set it to false to keep it consistent with previous behavior.

Reply:
Why is `RetryClusterClient` needed here and not elsewhere?

Reply:
In order to pass `&etcdserverpb.MemberListRequest{Linearizable: false}`